ghpages_deploy.sh

```diff
@@ -22,15 +22,15 @@ git config --global user.name "Yaydoc Bot"
 git config --global user.email "noreply+bot@example.com"
 
 GIT_SSH_URL=git@github.com:$USERNAME/$REPONAME.git
-git clone --quiet $GIT_SSH_URL gh-pages
+git clone --quiet $GIT_SSH_URL gh_pages
```

Comment:
Got an error while doing `rm -rf gh-pages` probably due to `-` in the name. Considering It's only a temporary folder, I renamed it.